TicketsDemo.Domain/DefaultImplementations/DefaultPriceCalculationStrategy.cs

```diff
@@ -14,30 +16,32 @@ public class DefaultPriceCalculationStrategy : IPriceCalculationStrategy
         private IRunRepository _runRepository;
         private ITrainRepository _trainRepository;
 
-        public DefaultPriceCalculationStrategy(IRunRepository runRepository, ITrainRepository trainRepository) {
+        public DefaultPriceCalculationStrategy(IRunRepository runRepository, ITrainRepository trainRepository)
+        {
             _runRepository = runRepository;
             _trainRepository = trainRepository;
         }
-
-        public List<PriceComponent> CalculatePrice(PlaceInRun placeInRun)
+       
+        public List<PriceComponent> CalculatePrice(PriceCalculationParametersDTO teaCoffeeBedParametrs)
```

Comment:
Ok, the name of the 'class' is changed to 'PriceCalculationParametersDTO' which is nice, but then the variable is still called 'teaCoffeeBedParametrs' which is quite confusing.